cluster/gce/upgrade.sh

```diff
@@ -102,6 +100,10 @@ function wait-for-master() {
 function prepare-upgrade() {
   ensure-temp-dir
   detect-project
+  # TODO(mbforbes): Do we still need get-password?
+  get-password
+  # TODO(mbforbes): Do we need get-bearer-token here?
```

Comment:
`KUBE_BEARER_TOKEN` is required by `build-kube-env`:
https://github.com/GoogleCloudPlatform/kubernetes/blob/9b2a0df811db637442daf93bcffefacaedea6f2a/cluster/gce/debian/helper.sh#L46

However, this comment in `configure-vm.sh`:
https://github.com/GoogleCloudPlatform/kubernetes/blob/9b2a0df811db637442daf93bcffefacaedea6f2a/cluster/gce/configure-vm.sh#L256

says that we don't need these data (`KUBE_PASSWORD`, `KUBE_USER`, `KUBE_BEARER_TOKEN`, `KUBELET_TOKEN`, `KUBE_PROXY_TOKEN`) except on init, and specifically not on an upgrade. However, it might be assuming that an upgrade happens in-place and retains the data that happens on cluster init; with the upgrade process introduced in this PR, that isn't true as we blow away the node.

The call to the function discussed in the previous paragraph, `create-salt-auth`, happens for both master and nodes:
https://github.com/GoogleCloudPlatform/kubernetes/blob/9b2a0df811db637442daf93bcffefacaedea6f2a/cluster/gce/configure-vm.sh#L456

so I don't know whether we can just remove these vars. @zmerlynn should probably chime in here.